apps/challenges/utils.py

```diff
@@ -6,3 +8,9 @@
 get_challenge_model = get_model_object(Challenge)
 
 get_challenge_phase_model = get_model_object(ChallengePhase)
+
+
+def get_file_content(file_path, mode):
```

Comment:
May I know why are we adding this function? I am still not able to infer its appropriate use.